src/components/sidebar/Sidebar.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <vs-sidebar square v-model="active" :open="$mq === 'xl' || $mq === 'lg'">
+        <vs-sidebar-item id="home"> Home </vs-sidebar-item>
+        <vs-sidebar-item id="market"> Market Overview </vs-sidebar-item>
+        <vs-sidebar-item id="Music"> Music </vs-sidebar-item>
+        <vs-sidebar-group>
+            <template #header>
+                <vs-sidebar-item arrow> Social media </vs-sidebar-item>
+            </template>
+
+            <vs-sidebar-item id="Instagram"> Instagram </vs-sidebar-item>
+            <vs-sidebar-item id="twitter"> Twitter </vs-sidebar-item>
+            <vs-sidebar-item id="Facebook"> Facebook </vs-sidebar-item>
```

Comment:
@PVillasana Here you are using capital letters to define ID, I think this can be confuse.